src/Filtering/BinaryMathematicalMorphology/ThinImage/Code.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import sys
+import itk
+
+PixelType = itk.UC
+Dimension = 2
+ImageType = itk.Image[PixelType, Dimension]
+
+if len(sys.argv) == 2:
+    image = itk.imread(sys.argv[1])
+
+else:
+    # Create an image
+    start = itk.Index[Dimension]()
+    start.Fill(0)
+
+    size = itk.Size[Dimension]()
+    size.Fill(100)
+
+    region = itk.ImageRegion[Dimension]()
+    region.SetIndex(start)
+    region.SetSize(size)
+
+    image = ImageType.New(Regions=region)
+    image.Allocate()
+    image.FillBuffer(0)
+
+    # Draw a 5 pixel wide line
+    for i in range(20, 80):
+        for j in range(50, 55):
+            index = itk.Index[Dimension]()
+            index[0] = i
+            index[1] = j
+            image.SetPixel(index, 255)
+
+    # Write Image
+    itk.imwrite(image, "input.png")
+
+image = itk.binary_thinning_image_filter(image)
+
+# Rescale the image so that it can be seen (the output is 0 and 1, we want 0 and 255)
+image = itk.rescale_intensity_image_filter(image, OutputMinimum=0, OutputMaximum=255)
```

Comment:
For a more Pythonic style, `OutputMinimum` -> `output_minimum` and `OutputMaximum` -> `output_maximum`.

Reply:
Done